packages/vscode-extension/PRERELEASE.md

```diff
@@ -4,6 +4,21 @@
 
 > Note: This changelog only includes the changes for the pre-release versions of Teams Toolkit. For the changelog of stable versions, please refer to the [Teams Toolkit Changelog](https://github.com/OfficeDev/TeamsFx/blob/dev/packages/vscode-extension/CHANGELOG.md).
 
+### June 12, 2024
+
+#### New Features
+
+- **Build AI Agent With Assistant API and Python**: Previously we have included the AI Assistant Bot app template to help you get started with building a GPT-like chat bot with AI capabilities using `Teams AI Library`. Now we have added a new AI Agent app template to help you build an AI agent with Assistant API and Python. This template showcases how to build an intelligent chat bot in Teams capable of helping users accomplish a specific task using natural language right in the Teams conversations, such as solving a math problem.
```

Comment:
just a comment, as I learned that MS is trying to avoid "agent" term in many places, so pls double check in this context we can use AI Agent.

Reply:
So far it's also used in docs: https://learn.microsoft.com/en-us/microsoftteams/platform/toolkit/build-an-ai-agent-in-teams?tabs=buildnew, we could align together with MS Learn. Do you if there is alternative for `agent`?